Assets/MixedRealityToolkit.Providers/WindowsMixedReality/Editor/ConfigurationChecker/WindowsMixedRealityConfigurationChecker.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.﻿
+
+using Microsoft.MixedReality.Toolkit.Utilities.Editor;
+using System.IO;
+using UnityEditor;
+
+namespace Microsoft.MixedReality.Toolkit.WindowsMixedReality
+{
+    /// <summary>
+    /// Class to perform checks for configuration checks for the Windows Mixed Reality provider.
+    /// </summary>
+    [InitializeOnLoad]
+    static class WindowsMixedRealityConfigurationChecker
```

Comment:
Should we explicitly add `private` here?

Reply:
see earlier comment re: language support